src/code/audio/audio_data.c

```diff
@@ -0,0 +1,895 @@
+
+    // Frequency of 8
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  -32767, -24575, -16383, -8191,
+};
+
+s16 gTriangleWaveSample[] = {
+    // Frequency of 1
+    0,      2047,   4095,   6143,   8191,   10239,  12287,  14335,
+    16383,  18431,  20479,  22527,  24575,  26623,  28671,  30719,
+    32767,  30719,  28671,  26623,  24575,  22527,  20479,  18431,
+    16383,  14335,  12287,  10239,  8191,   6143,   4095,   2047,
+    0,      -2047,  -4095,  -6143,  -8191,  -10239, -12287, -14335,
+    -16383, -18431, -20479, -22527, -24575, -26623, -28671, -30719,
+    -32767, -30719, -28671, -26623, -24575, -22527, -20479, -18431,
+    -16383, -14335, -12287, -10239, -8191,  -6143,  -4095,  -2047,
+
+    // Frequency of 2
+    0,      4095,   8191,   12287,  16383,  20479,  24575,  28671,
+    32767,  28671,  24575,  20479,  16383,  12287,  8191,   4095,
+    0,      -4095,  -8191,  -12287, -16383, -20479, -24575, -28671,
+    -32767, -28671, -24575, -20479, -16383, -12287, -8191,  -4095,
+    0,      4095,   8191,   12287,  16383,  20479,  24575,  28671,
+    32767,  28671,  24575,  20479,  16383,  12287,  8191,   4095,
+    0,      -4095,  -8191,  -12287, -16383, -20479, -24575, -28671,
+    -32767, -28671, -24575, -20479, -16383, -12287, -8191,  -4095,
+
+    // Frequency of 4
+    0,      8191,   16383,  24575,  32767,  24575,  16383,  8191,
+    0,      -8191,  -16383, -24575, -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  32767,  24575,  16383,  8191,
+    0,      -8191,  -16383, -24575, -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  32767,  24575,  16383,  8191,
+    0,      -8191,  -16383, -24575, -32767, -24575, -16383, -8191,
+    0,      8191,   16383,  24575,  32767,  24575,  16383,  8191,
+    0,      -8191,  -16383, -24575, -32767, -24575, -16383, -8191,
+
+    // Frequency of 8
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+    0,      16383,  32767,  16383,  0,      -16383, -32767, -16383,
+};
+
+s16 gSineWaveSample[] = {
+    // Frequency of 1
```

Comment:
Would these be harmonics instead of frequency, ex: 1st Harmonic, 2nd Harmonic, 4th Harmonic?

Reply:
That's a nicer way to phrase it, yeah